src/Analyzers/CSharp/Tests/UseImplicitOrExplicitType/UseExplicitTypeTests.cs

```diff
@@ -470,6 +476,8 @@ void Method(Program? x)
             await TestInRegularAndScriptAsync(before, after, options: ExplicitTypeExceptWhereApparent());
         }
 
+#if !CODE_STYLE // Tests depending on product code which need new nullable APIs are disabled for Code Style layer
```

Comment:
This is the same issue that @CyrusNajmabadi also saw in his refactoring PR. Our product code currently depends on very recently added compiler APIs for nullable annotations, but these are not available in 3.3.0 Microsoft.CodeAnalysis package that CodeStyle layer references. I will file a tracking issue to enable these tests in CodeStyle layer, which would need also removing the `#if !CODE_STYLE` statements in product code that invokes these APIs.

Reply:
can we add the tracking issue into the code here?

Reply:
that way, whne it is fixed, we know what to go back and update?